pyanaconda/modules/payloads/kickstart.py

```diff
@@ -279,6 +279,7 @@ class PayloadKickstartSpecification(KickstartSpecification):
         "nfs": COMMANDS.NFS,
         "ostreecontainer": COMMANDS.OSTreeContainer,
         "ostreesetup": COMMANDS.OSTreeSetup,
+        "bootc": COMMANDS.Bootc,
```

Comment:
I don' think you need to add it to payloads module right now as there is no code to process it.

Reply:
If I got the test right - anaconda is comparing all bootc command with expected list. So by having `bootc` here I am adding it to the expected list.

Reply:
Yes, but it is not expected until used. It should be adopted by the module with the logic how to parse and process it. This command is not used by Anaconda yet.

Reply:
I see the test explicitly requires to have all the commands assigned somewhere in Anaconda.
In that case this is probably correct approach.